### ISSUE TITLE
Error when overriding keymap for process view

### ISSUE THREAD

**Issue by psibi** [pvolok/mprocs#35]
I have the following `mprocs.yaml` where I try to override the keymap:

```
keymap_procs:
  <j> : { "C-n": next-proc }
procs:
  docker compose:
    cmd: ["docker-compose", "up"]
    autostart: false
  compose down:
    cmd: ["docker-compose", "down", "-v"]
    autostart: false
```

When I invoke `mprocs`, I get the following error:

```
❯ mprocs
Error: [local config] missing field `c`
```

**Comment by pvolok** [comment]
It should be:
```yaml
keymap_procs:
  <j>: { c: next-proc }
```

If you want to bind it to `C-n`:
```yaml
keymap_procs:
  <C-n>: { c: next-proc }
```

**Comment by psibi** [comment]
Thanks, that works. But there is one issue: The `Help` area in the TUI isn't updated to reflect the new keybindings.

**Comment by pvolok** [comment]
Yes, that's a known issue. I'll fix it some time in the future.

**Comment by pvolok** [comment]
Now help windows reflect actual key bindings, including overrides. Included in mprocs 0.6. Thanks for your PR, it helped me to come up with a solution.

**Comment by psibi** [comment]
Nice! Thank you.
